.ci/docker/node-puppeteer/Dockerfile

```diff
@@ -1,14 +1,33 @@
-#Dockerfile for docker.elastic.co/observability-ci/node-puppeteer:12
-# When changing Node version, please build the Docker image for RUM here:
-#   https://apm-ci.elastic.co/job/apm-shared/job/apm-docker-images-pipeline/build?delay=0sec
-# Remember checking the 'rum' parameter
 ARG NODEJS_VERSION
-FROM node:${NODEJS_VERSION}
+FROM ubuntu:jammy
```

Comment:
Did you talk to the Robots team about using this custom image? 

The other option is to fix our obserability ci image which is from our shared pipeline.

Reply:
I didn't mention anything explicitly because, in the other image, we were using `ubuntu:bionic` which is another custom "image".

I can share this with Robots, and discuss if it would be required to align approaches in future iterations

Reply:
It's fine to use any distro base image from a trusted provider.